pymc/step_methods/metropolis.py

```diff
@@ -140,22 +140,22 @@ def __init__(
 
         Parameters
         ----------
-        vars: list
+        vars : list
             List of value variables for sampler
-        S: standard deviation or covariance matrix
+        S : standard deviation or covariance matrix
             Some measure of variance to parameterize proposal distribution
-        proposal_dist: function
+        proposal_dist : function
             Function that returns zero-mean deviates when parameterized with
             S (and n). Defaults to normal.
-        scaling: scalar or array
+        scaling : scalar or array
             Initial scale factor for proposal. Defaults to 1.
-        tune: bool
+        tune : bool
             Flag for tuning. Defaults to True.
-        tune_interval: int
+        tune_interval : int
             The frequency of tuning. Defaults to 100 iterations.
-        model: PyMC Model
+        model : PyMC Model
             Optional model for sampling step. Defaults to None (taken from context).
-        mode: string or `Mode` instance.
+        mode : string or `Mode` instance.
```

Comment:
I don't really know what `Mode` is so it would be nice to have it as a link, unfortunately I can't give a suggestion unless I know where it should point to. Hopefully someone else can comment on this

Reply:
It's a pytensor thing... Not sure we have a more precise signature anywhere. The class is https://github.com/pymc-devs/pytensor/blob/a76172ee6a9753266150fc7bac4dd66906967a26/pytensor/compile/mode.py#L275

But it can also be one of a predefined set of strings :D